src/main/java/hw10/FileLogger.java

```diff
@@ -0,0 +1,75 @@
+package hw10;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+public class FileLogger {
+    private final FileLoggerConfiguration configuration;
+    private final DateTimeFormatter fileDateFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public FileLogger(FileLoggerConfiguration configuration) {
+        this.configuration = configuration;
+    }
+
+    public void debug(String message) {
+        Objects.requireNonNull(message);
+        log(LoggingLevel.DEBUG, message);
+    }
+
+    public void info(String message) {
+        Objects.requireNonNull(message);
+        log(LoggingLevel.INFO, message);
```

Comment:
The same here, this should be wrapped with `if (isInfoEnabled())` then do logging operation, otherwise do nothing.